platforms/avr/drivers/ws2812.c

```diff
@@ -23,6 +23,7 @@
 #include "ws2812.h"
 #include <avr/interrupt.h>
 #include <avr/io.h>
+#include <pin_defs.h>
```

Comment:
```suggestion
#include "pin_defs.h"
#include <avr/interrupt.h>
#include <avr/io.h>
```

Reply:
`pin_defs.h` uses `_SFR_IO8()`, which `avr/io.h` defines inside. Therefore, it is natural to load `avr/io.h` first.

Reply:
The suggestion was more to somewhat follow the current repo conventions.

The include block could be,
```
#include <avr/interrupt.h>
#include <avr/io.h>
#include <util/delay.h>
#include "ws2812.h"
#include "pin_defs.h"
```
important part is that we currently do not use `<>` for qmk headers

Reply:
> The include block could be,
> 
> ```
> #include <avr/interrupt.h>
> #include <avr/io.h>
> #include <util/delay.h>
> #include "ws2812.h"
> #include "pin_defs.h"
> ```

OK, I do this.